specification/src/main/asciidoc/platform/RelatedDocuments.adoc

```diff
@@ -4,149 +4,85 @@ This specification refers to the following
-_http://jcp.org/en/jsr/detail?id=77_ .
+Jakarta Management Specification, Version 1.1. Available at: <link needed>.
 
-Java™ Platform, Enterprise Edition Deployment
-Specification 1.2 (Java EE Deployment specification). Available at
-_http://jcp.org/en/jsr/detail?id=88_ .
+Jakarta Deployment Specification, Version 1.2. Available at: <link needed>.
 
-Java™ Authorization Service Provider Contract
-for Containers 1.5 (JACC specification). Available at
-_http://jcp.org/en/jsr/detail?id=115_ .
+Jakarta Authorization Specification, Version 1.5. Available at: <link needed>.
 
-Java™ Authentication Service Provider Interface
-for Containers 1.1 (JASPIC specification). Available at
-_http://jcp.org/en/jsr/detail?id=196_ .
+Jakarta™ Authentication Specification, Version 1.1. Available at: <link needed>.
 
-Java™ EE Security API 1.0. Available at
-_http://jcp.org/en/jsr/detail?id=375_ .
+Jakarta™ Security Specification, Version 1.0. Available at: <link needed>.
 
-Debugging Support for Other Languages 1.0.
-Available at _http://jcp.org/en/jsr/detail?id=45_ .
+Jakarta™ Debugging Support for Other Languages Specification, Version 1.0. Available at: <link needed>.
 
-Standard Tag Library for JavaServer Pages 1.2
-(JSTL specification). Available at _http://jcp.org/en/jsr/detail?id=52_
-.
+Jakarta™ Standard Tag Library Specification, Version 1.2. Available at: <link needed>.
 
-Web Services Metadata for the Java Platform
-2.1. Available at _http://jcp.org/en/jsr/detail?id=181_ .
+Jakarta Web Services Metadata Specification, Version 2.1. Available at: <link needed>.
 
-JavaServer Faces 2.3 (JSF specification).
-Available at _http://jcp.org/en/jsr/detail?id=372_ .
+Jakarta™ Server Faces Specification, Version 2.3. Available at: <link needed>.
 
-Java Persistence 2.2 (Java Persistence
-specification). Available at _http://jcp.org/en/jsr/detail?id=338_ .
+Jakarta™ Persistence Specification, Version 2.2. Available at: <link needed>.
 
-Bean Validation 2.0 (Bean Validation
-specification). Available at _http://jcp.org/en/jsr/detail?id=380_ .
+Jakarta™ Bean Validation Specification, Version 2.0. Available at: <link needed>.
 
-Managed Beans 1.0 (Managed Beans
-specification). Available at _http://jcp.org/en/jsr/detail?id=316_ .
+Jakarta™ Managed Beans Specification, Version 1.0. Available at: <link needed>.
 
-Interceptors 1.2 _rev A_ (Interceptors
-specification). Available at _http://jcp.org/en/jsr/detail?id=318_ .
+Jakarta™ Interceptors Specification, Version 1.2. Available at: <link needed>.
 
-Contexts and Dependency Injection for the Java
-EE Platform 2.0 (CDI specification). Available at
-_http://jcp.org/en/jsr/detail?id=365_ .
+Jakarta™ Contexts and Dependency Injection Specification, Version 2.0. Available at: <link needed>.
 
-Dependency Injection for Java 1.0 (DI
-specification). Available at _http://jcp.org/en/jsr/detail?id=330_ .
+Dependency Injection for Java 1.0 (DI specification). Available at http://jcp.org/en/jsr/detail?id=330
```

Comment:
I believe DI is going to be produced as part of CDI...  For now, this is okay, but we'll have to watch this space.

Reply:
Yes, the CDI project will be producing the DI spec.